app/konnect/dev-portal/create-dev-portal.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Create a Dev Portal
+---
+
+The {{site.konnect_short_name}} Dev Portal is a customizable website for developers to locate, access, and consume API services. You can create both public and private Dev Portals, depending on your use case. With a public Dev Portal, anyone with the URL can view the APIs you add to your Dev Portal. With a private Dev Portal, only those who have signed in to the Dev Portal can see the APIs you add to your Dev Portal. Additionally, when you configure private Dev Portals, you can also allow developers to register apps with your APIs. You can also publish your APIs to both a private Dev Portal as well as a public one, such as if you want to have one Dev Portal for internal developers at your company as well as allow external developers at other companies to create apps with some of your APIs.
+
+## Prerequisites
+
+Configure an authentication strategy if you plan to use one. This auth strategy will be how developers authenticate when they use your APIs. <!-- tried searching for a link to a doc for this, is it just setting up an auth plugin?-->
+
+## Steps
+{% navtabs %}
+{% navtab Konnect UI %}
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click **Dev Portal** on the top right.
+1. In the Create Dev Portal dialog, configure your Dev Portal settings:
+    1. Enter general information for your Dev Portal, including a name and optional description.
+    1. Select if you want the Dev Portal to be private or public. Developers can only register apps to private Dev Portals.
+    1. Private Dev Portal only: Configure the following settings:
+        * **Auto Approve Developers**: If you enable this setting, this will allow developers to automatically be approved when they sign up for your private Dev Portal. If you don't enable this, you must configure single sign-on or manually approve developers as they register. 
+        * **Auto Approve Applications**: If you enable this setting, apps that developers create with your APIs will be automatically approved. If you don't enable this, you must manually approve or deny application registrations.
+        * **Portal RBAC**: You must enable this setting if you plan to allow developers to register with your Dev Portal. Portal RBAC allows you to assign developers to teams and roles that determine if they can only view or view and consume the APIs in your Dev Portal.
```

Comment:
```suggestion
        * **Portal RBAC**: Portal RBAC allows you to assign developers to teams and roles that determine if they can only view or view and consume the APIs in your Dev Portal.
```

Reply:
this "You must enable this setting if you plan to allow developers to register with your Dev Portal. " is not true